lib/tools.js

```diff
@@ -694,5 +870,202 @@ module.exports = {
                 res.json(data);
             }
         };
+    },
+
+    async generateAPiDocs(routes) {
+        let docs = `
+openapi: 3.0.0
```

Comment:
openapi spec doesn't have to be yaml, it can be JSON. I just used yaml because it was easier to edit by hand. I think it would make more sense here to just work with js objects/arrays normally and output a json file at the end right? Seems like it would be much more rebust than appending to a yaml string.

Reply:
Yes. You are correct. In the upcoming commits I am planning to make it all JSON, that way it's more standardized and more easily maintainable! Also more robust as you have pointed out